splinter/driver/djangoclient.py

```diff
@@ -141,17 +141,17 @@ def htmltree(self):
         try:
             return self._html
         except AttributeError:
-            self._html = lxml.html.fromstring(self.html.decode('utf-8'))
+            self._html = lxml.html.fromstring(self.html)
             return self._html
 
     @property
     def title(self):
         html = self.htmltree
-        return html.xpath('//title')[0].text_content().strip().encode('utf-8')
+        return html.xpath('//title')[0].text_content().strip()
 
     @property
     def html(self):
-        return self._response.content
+        return self._response.content.decode(self._response.charset)
```

Comment:
I think there is a regression here, Django 1.7 TemplateResponse doesn't have a "charset" property. This seems to be [fixed](https://github.com/cobrateam/splinter/blob/master/splinter/driver/djangoclient.py#L172) on `master` but makes 0.7.2 unusable with Django 1.7.